R/staggregate.R

```diff
@@ -793,7 +1041,7 @@ staggregate_degree_days <- function(data, overlay_weights, time_agg = "month", s
 
   # Create function to calculate degree days
   calc_deg_days <- function(x){
-    clim_table <- raster::values(clim_rast)
+    clim_table <- terra::values(clim_rast)
```

Comment:
Same issue as before, does this need to be done everytime?

Additionally, nested ifelse's like below are ugly. Perfect uusecase for case_when or ssomething more readable

Reply:
Agree that nested ifelse's are ugly, but we tried using case_when and this broke something. Happy to revisit though when making efficiency improvements